app/models/book.rb

```diff
@@ -1,3 +1,5 @@
 class Book < ApplicationRecord
   validates :genere, :author, :image, :title, :publisher, :year, presence: true
+
+  has_many :rents
```

Comment:
What should Rails do with the associated Rents when a Book is deleted? Check `4.3.2.5 :dependent` section from [here](http://guides.rubyonrails.org/association_basics.html#has-many-association-reference).